content/docs/start/model-management/model-cicd.md

```diff
@@ -74,12 +74,12 @@ specific format.
 This also means that we can create CICD actions in our Git repository which will
```

Comment:
WDYT about breaking this section into subsections, like:
1. Trigger the workflow
2. Parse the model registry info
3. Check whether to deploy
4. Deploy model

Reply:
Or we can combine 1+2 into "setup" and 3+4 into "deploy." The level of detail is great in this section, but let's see if we can make it less overwhelming. At least half the workflow is boilerplate that can be copied verbatim, and we might not need to break that part down so much, or at least we can clarify that it's not critical to understand every line of that part.